UVtools.Core/Layer/LayerManager.cs

```diff
@@ -1493,7 +1493,22 @@ void CacheLayers(uint layerIndex, bool direction)
                             var suctionTrapArea = EmguContours.GetContourArea(trap);
                             if (suctionTrapArea < minimumSuctionArea) continue;
                             var rect = CvInvoke.BoundingRectangle(trap[0]);
-                            AddIssue(new LayerIssue(this[layerIndex], LayerIssue.IssueType.SuctionCup, trap.ToArrayOfArray(), rect) { Area = suctionTrapArea });
+                            var newIssue = new LayerIssue(this[layerIndex], LayerIssue.IssueType.SuctionCup, trap.ToArrayOfArray(), rect) { Area = suctionTrapArea };
+
+                            if (layerIndex < LayerCount - 1 && suctionTraps[layerIndex + 1] is not null)
```

Comment:
Can this be moved to the repair method instead? and discard the ChildIssues and ParentIssue only used by this issue?

Reply:
This should be addressed by most recent commit.